Sources/Jetworking/Client/ResponseHandler.swift

```diff
@@ -0,0 +1,77 @@
+import Foundation
+
+enum ResponseHandler {
+    typealias WrappedCompletion<ResponseType> = (HTTPURLResponse, Data?, Decoder, @escaping Client.RequestCompletion<ResponseType>) -> () -> Void
+    enum CompletionWrapper {
+        static func void<ResponseType>(
```

Comment:
A function should contain a verb what it is doing. Maybe we can use something like `wrapCompletionWithVoidResponse` or similar. What do you think?

Reply:
Yes, but I would consider the `void` and `decodable` functions alike factory functions (like a `default` function to instantiate a default object of some type). Since these functions are used to obtain a `WrappedCompletion` signature they are a "factory" for these signatures. Compare it also to the evaluate calls in `Client`:
```
ResponseHandler.evaluate(
    data: data,
    urlResponse: urlResponse,
    error: error,
    endpoint: endpoint,
    configuration: self.configuration,
    completionWrapper: ResponseHandler.CompletionWrapper.decodable,
    completion: completion
)
```

Reply:
If they are factory functions they should start with `make` for example `makeVoidCompletionWrapper` - Follow Swift API guidelines:

https://swift.org/documentation/api-design-guidelines/#strive-for-fluent-usage

Reply:
These methods are now renamed